### PR TITLE
Fix Nawawi Arabic Hadith text formatting of links

### LINKED ISSUE #71: Arabic Hadith Formatting Bug
Some Arabic hadith, I think just the 40 Hadith of Nawawi, have a reference using the [] () syntax and it doesn't work properly because of the mix of the Arabic text and English URL. However, the English version works perfectly.
image

### PR DESCRIPTION
**Pull request by itzmk21**
Closes #71 

Gets the [] and (link...) and removes them, then adds them back properly at the end of the Hadith text (if it applies to that Hadith in the Nawawi Arabic Collection since not all hadith have reference links and thus are left alone).

image
image
image

Sometimes on some hadith there's extra dots and commas at the end before the embed links which are hard to remove since every Hadith is different.

image

Tested on all 40 Ahaadith.

### CODE REVIEW

File: hadith/hadith.py [hunk 1]
```diff
@@ -153,6 +153,24 @@ def process_hadith(self, hadith_list):
                     pass
 
         self.text = self.format_hadith_text(self.text, self.lang)
+        
+        if self.lang == 'ar' and self.collection == "forty":  # if ahadith and is nawawi collection
```
Comment (galacticwarrior9) [inline hadith/hadith.py:157]: Can you add a link to the issue here so that it is clearer what the purpose of the code is for future contributors?

Comment (itzmk21) [reply hadith/hadith.py:157]: Yep, sure thing

File: miscellaneous/TopGG.py [hunk 2]
```diff
@@ -13,7 +13,7 @@ class TopGG(commands.Cog):
     def __init__(self, bot):
         self.bot = bot
         self.token = config['APIs']['top.gg']
-        self.dblpy = topgg.DBLClient(self.bot, self.token, autopost=True, post_shard_count=True)
+        # self.dblpy = topgg.DBLClient(self.bot, self.token, autopost=True, post_shard_count=True)
```
Comment (galacticwarrior9) [inline miscellaneous/TopGG.py]: Make sure to uncomment this

Comment (itzmk21) [reply miscellaneous/TopGG.py]: Oh man, sorry. Didn't realise I added that file too for that commit.

**Pull request merged**